### PR TITLE
Clearqfbwjb

### PR DESCRIPTION
**Pull request by wjblanke**

### CODE REVIEW

**Comment by wjblanke** [comment]
Changed

> a shorter way of saying this is just:
> 
> ```
> qfb fr{}, fr2{}, fr3{};
> ```

File: src/nucomp.h [hunk 1]
```diff
@@ -193,7 +193,7 @@ void qfb_nucomp(qfb_t r, const qfb_t f, const qfb_t g, mpz_t& D, mpz_t& L)
 
 // a = b * c
 void nucomp_form(form &a, form &b, form &c, integer &D, integer &L) {
-    qfb fr, fr2, fr3;
```
Comment (arvidn) [inline src/nucomp.h:196]: actually, it it occurred to me this morning; do we ever want to declare `qfb` objects that are uninitialized? If not, we should just do this in its constructor and rest assured we won't have this problem again.

**Comment by arvidn** [comment]
I'll test this on raspberry PI

**Comment by arvidn** [comment]
this does not solve the problem. I have a debug build on RPi now though, so I'll investigate

**Comment by arvidn** [comment]
https://github.com/Chia-Network/chiavdf/pull/79

**Pull request closed without merging**